From 7be24a34b3c4e9294b9093b5184e290061956baf Mon Sep 17 00:00:00 2001
From: Jie Liu <liujie375@h-partners.com>
Date: Mon, 24 Oct 2022 09:34:57 +0800
Subject: [PATCH] sched:Open the kernel configuration for cluster.

kunpeng inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5W44S

----------------------------------------------------------------------

In the past configuration, CONFIG_SCHED_CLUSTER was not set. Now, we need
to open the configuration.

Signed-off-by: Jie Liu <liujie375@h-partners.com>
Signed-off-by: Jiang Yi <jiangyi38@hisilicon.com>
Signed-off-by: Xue Sinian <xuesinian@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index ded320901f60..d201943b3024 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -419,6 +419,7 @@ CONFIG_ARM64_PA_BITS_48=y
 CONFIG_ARM64_PA_BITS=48
 # CONFIG_CPU_BIG_ENDIAN is not set
 CONFIG_SCHED_MC=y
+CONFIG_SCHED_CLUSTER=y
 # CONFIG_SCHED_SMT is not set
 CONFIG_NR_CPUS=1024
 CONFIG_HOTPLUG_CPU=y
@@ -1770,6 +1771,7 @@ CONFIG_NET_DEVLINK=m
 CONFIG_MAY_USE_DEVLINK=m
 CONFIG_PAGE_POOL=y
 CONFIG_FAILOVER=m
+CONFIG_ETHTOOL_NETLINK=y
 CONFIG_HAVE_EBPF_JIT=y
 
 #
@@ -1905,7 +1907,6 @@ CONFIG_MTD_PHYSMAP_OF=m
 #
 # CONFIG_MTD_PMC551 is not set
 # CONFIG_MTD_DATAFLASH is not set
-# CONFIG_MTD_M25P80 is not set
 # CONFIG_MTD_MCHP23K256 is not set
 # CONFIG_MTD_SST25L is not set
 # CONFIG_MTD_SLRAM is not set
@@ -3216,7 +3217,7 @@ CONFIG_I2C_SLAVE_EEPROM=m
 CONFIG_SPI=y
 # CONFIG_SPI_DEBUG is not set
 CONFIG_SPI_MASTER=y
-# CONFIG_SPI_MEM is not set
+CONFIG_SPI_MEM=y
 
 #
 # SPI Master Controller Drivers
@@ -5793,9 +5794,9 @@ CONFIG_QCOM_L3_PMU=y
 CONFIG_XGENE_PMU=y
 CONFIG_ARM_SPE_PMU=y
 CONFIG_HISI_PMU=m
+CONFIG_HISI_PCIE_PMU=m
 CONFIG_HISI_L3T_PMU=m
 CONFIG_HISI_LPDDRC_PMU=m
-CONFIG_HISI_PCIE_PMU=m
 CONFIG_RAS=y
 
 #
@@ -6189,9 +6190,10 @@ CONFIG_CRYPTO_RNG_DEFAULT=y
 CONFIG_CRYPTO_AKCIPHER2=y
 CONFIG_CRYPTO_AKCIPHER=y
 CONFIG_CRYPTO_KPP2=y
+CONFIG_CRYPTO_KPP=m
 CONFIG_CRYPTO_ACOMP2=y
 CONFIG_CRYPTO_RSA=y
-# CONFIG_CRYPTO_DH is not set
+CONFIG_CRYPTO_DH=m
 # CONFIG_CRYPTO_ECDH is not set
 CONFIG_CRYPTO_MANAGER=y
 CONFIG_CRYPTO_MANAGER2=y
@@ -6456,6 +6458,7 @@ CONFIG_FONT_SUPPORT=y
 # CONFIG_FONTS is not set
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
+CONFIG_SG_SPLIT=y
 CONFIG_SG_POOL=y
 CONFIG_ARCH_HAS_SG_CHAIN=y
 CONFIG_ARCH_HAS_PMEM_API=y
-- 
2.27.0

